crypto/aes/aesgcm.h

```diff
@@ -0,0 +1,233 @@
+/*
+---------------------------------------------------------------------------
+Copyright (c) 1998-2010, Brian Gladman, Worcester, UK. All rights reserved.
+
+The redistribution and use of this software (with or without changes)
+is allowed without the payment of fees or royalties provided that:
+
+  source code distributions include the above copyright notice, this
+  list of conditions and the following disclaimer;
+
+  binary distributions include the above copyright notice, this list
+  of conditions and the following disclaimer in their documentation.
```

Comment:
This comment is not directly related to this pull request as the header was already present in our codebase.
However, does our binary distribution comply with this notice?

Reply:
@matejcik?

Reply:
https://satoshilabs.slack.com/archives/CKN5C51CJ/p1711375279588959?thread_ts=1711359561.629369&cid=CKN5C51CJ